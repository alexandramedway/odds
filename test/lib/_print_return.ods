do x = print("Hello, world!")
do y = print(42)
do z = print([1, 2, 3])

do print(x)
do print(y)
do print(z)
