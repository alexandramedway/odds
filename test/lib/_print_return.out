Hello, world!
42
[1, 2, 3]
Hello, world!
42
[1, 2, 3]
